### PR TITLE
Integrity hotkey

### PR DESCRIPTION
**Pull request by mairdl**
The "Check integrity"-action seems to be in need of a hotkey. (See #1908) I used `CTRL + F8` for this action, because F8 is currently used to Cleanup entries. These 2 functions seem to be related so it would make sense to group them.

- [x] Change in CHANGELOG.md described
- [x] Manually tested changed features in running JabRef
- [x] If you changed the localization: Did you run `gradle localizationUpdate`?
- [x] internal qs

### CODE REVIEW

File: src/main/java/net/sf/jabref/gui/keyboard/KeyBinding.java [hunk 1]
```diff
@@ -13,6 +13,7 @@
     BACK("Back", Localization.lang("Back"), "alt LEFT"),
     CLEANUP(
             "Cleanup", Localization.lang("Cleanup entries"), "F8"),
+    CHECK_INTEGRITY("Check integrity", Localization.lang("Check integrity"), "ctrl F8"),
```
Comment (koppor) [inline src/main/java/net/sf/jabref/gui/keyboard/KeyBinding.java]: What was our last consensus about showing menu entries in the hotkey dialog? I think, I opted for using `Localization.lang`, but we might have decided to re-translate these entries. @boceckts Do you remember? I think, we had the discussion. @tobiasdiez Is that documented in our CodeHowTo in the Wiki?

File: CHANGELOG.md [hunk 2]
```diff
@@ -66,6 +66,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - [koppor#61](https://github.com/koppor/jabref/issues/61) Display gray background text in "Author" and "Editor" field to assist newcomers
 - Updated Vietnam translation
 - Added greyed-out suggestion for `year`/`date`/`url` fields
+- [#1908] (https://github.com/JabRef/jabref/issues/1908) Add a shortcut for check integrity
```
Comment (koppor) [inline CHANGELOG.md]: Why is the new shortcut not listed here?

File: src/main/java/net/sf/jabref/gui/keyboard/KeyBinding.java (hunk 1 above)
Comment (boceckts) [inline src/main/java/net/sf/jabref/gui/keyboard/KeyBinding.java]: The thing I rember was about the keybinding **category** where we should use `Localization.menuTitle` as you wrote in https://github.com/JabRef/jabref/pull/1390#issuecomment-238339367. This affects only the javafx branch so far. Other than that I don't remeber a discussion about the menu entries in the hotkey dialog. :thinking:

Comment (mairdl) [inline src/main/java/net/sf/jabref/gui/keyboard/KeyBinding.java]: In the `KeyBinding.java` only `Localization.lang` gets used. So I guess it must be right.

File: src/main/resources/l10n/JabRef_fa.properties [hunk 3]
```diff
@@ -2300,3 +2300,5 @@ Do_you_want_to_recover_the_database_from_the_backup_file?=Do_you_want_to_recover
 Firstname_Lastname=
 
 This_might_be_caused_by_reaching_the_traffic_limitation_of_Google_Scholar_(see_'Help'_for_details).=
+
+Check_integrity=?????_???????
```
Comment (stefan-kolb) [inline src/main/resources/l10n/JabRef_fa.properties]: This doesn't look right here?

**Comment by koppor** [comment]
Thanx. So, please, no copy'n'paste solutions here and useage of .menuTitle

**Comment by mairdl** [comment]
I changed it to menuTitle and removed the unused keys.

**Pull request merged**